### ISSUE TITLE
Misdescription or bug in tutorial section 3.5

### ISSUE THREAD

**Issue by zpah** [helix-editor/helix#6231]
Version: helix-22.12

Tutorial section 3.5 give wrong result.

Quote from section 3.5:
1. Move the cursor to the line marked '-->' below.
2. Move to the F of FOO and type v2w to select the two words.
3. Type d to remove the two words.

Sample line is:
' --> Remove the FOO BAR distracting words BAZ BIZ from this line.'

Expected result:
' --> Remove the distracting words BAZ BIZ from this line.'
Actual:
' --> Remove the FOOdistracting words BAZ BIZ from this line.'

Steps to reproduce:
1) Type 324G move cursor at column 1 in line 324.
2) Type 4wb (as described in Section 3.4) move cursor to the F and next type v2w (wrong selects ' BAR ').
3) Type d erases ' BAR '.

**Comment by JustinLovinger** [comment]
Relevant comment: <https://github.com/helix-editor/helix/pull/1570#issuecomment-1460191286>. With the current behavior, the correct key sequence for this tutorial is `wvw` or `;v2w`.

**Comment by zpah** [comment]
Thank you for response. It works right.

It would be nice to correct tutorial and explain that (w, b, e) commands is not suitable for motion and is not equivalent to (j, k, l, h) commands but used for selection only.

**Comment by JustinLovinger** [comment]
`w`, `b`, and `e` can be used for movement. The issue is an inconsistency where only *some* motions select. If `2w` selected two words, this tutorial could just be `2w` without the `;v`.

**Comment by zpah** [comment]
I agree with you about inconsistency of `w` and `2w` (or any `[count] w`) selections.

But what about motions _without_ selections?

For example, `j`, `k`, `l`, `h` does _not select_ (motion only), but `w`, `b`, `e` _selects_ (here their mutual inconsistency).
In other words, how to move through text objects (words, paragraphs, ...) _without_ selections (`h`, `j`, or vim-style `w`, `b`, `e`) ?

**Comment by JustinLovinger** [comment]
`j`, `k`, `l`, `h` selects the character moved over, a one-character selection. In Helix, something is always selected.

**Comment by zpah** [comment]
Thanks for your explanation. I was missing it.

But if you`re proposing that `2w` should select _two words_, then and `2l`, `2h`, `2k`, `2j` should also select _two characters_ (for consistency), shouldn`t it?

**Comment by JustinLovinger** [comment]
Yes, for consistency, all motions should select characters traversed over. There is an old unresolved issue about this, <https://github.com/helix-editor/helix/issues/536>, and keybinding are still being discussed in general.
